modules/virt-updating-workloads.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-updating-workloads_{context}"]
+= Updating virtualization workloads
+
+When you upgrade {VirtProductName}, VirtualMachineInstance (VMI) workloads, such as `virt-launcher` pods, update automatically by default. You can configure workload update strategies or opt out of workload updates by editing the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* To use live migration as an update method, you must first enable live migration in the cluster.
++
+[NOTE]
+====
+If a `VirtualMachineInstance` CR contains `evictionStrategy: LiveMigrate` and the VMI does not support live migration, the VMI will not update.
+====
+
+.Procedure
+
+. To open the `HyperConverged` CR in your default editor, run the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `workloadUpdateStrategy` stanza of the `HyperConverged` CR. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  workloadUpdateStrategy:
+    workloadUpdateMethods: <1>
+    - LiveMigrate <2>
+    - Evict <3>
+    batchEvictionSize: 10 <4>
+    batchEvictionInterval: "1m" <5>
+----
+<1> The methods that can be used to perform automated workload updates. The supported values are `LiveMigrate` and `Evict`, both of which are enabled by default. When both are enabled, updates use `LiveMigrate` unless a VMI does not support live migration, in which case `Evict` is used. You can disable automatic workload updates by leaving the array empty (`workloadUpdateMethods: []`).
+<2> The least disruptive update method. VMIs that support live migration are updated by migrating the virtual machine (VM) guest into a new pod with the updated components enabled. If `LiveMigrate` is the only workload update method in the list, VMIs that do not support live migration are not disrupted or updated.
+<3> A disruptive method that shuts down VMI pods during upgrade. `Evict` is the only update method available if live migration is not enabled in the cluster. If a VMI is controlled by a `VirtualMachine` object that has `runStrategy: always` configured, a new VMI is created in a new pod with updated components.
+<4> The number of VMIs that can be forced to be updated at a time. The default value is `10`.
+<5> The interval to wait before issuing the next batch of evictions.
```

Comment:
Since for the other two, default value is being explicitly called out, should the same happen here as well?